docs/core/deploying/native-aot/libraries/ios-like-native-libraries-with-nativeaot.md

```diff
@@ -0,0 +1,261 @@
+---
+title: Building native libraries for iOS-like platforms
+description: How to build and consume native libraries with Native AOT for iOS-like platforms
+author: ivanpovazan
+ms.author: ivanpovazan
+ms.date: 11/11/2024
+---
+
+# Building native libraries for iOS-like platforms
+
+Starting from .NET 9, Native AOT officially supports publishing .NET class libraries for iOS-like platforms.
+This enables users to create self-contained native libraries that can be consumed from iOS, Mac Catalyst and tvOS applications.
```

Comment:
I assume that Objective-C interop is not available for native libraries built this way. Is that correct? Should we mention it here?

Reply:
Thank you, fixed.